sdk/maps/maps-search/tests.yml

```
@@ -0,0 +1,13 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/maps-search"
+      ServiceDirectory: maps
+      SupportedClouds: 'Public,Canary'
+      EnvVars:
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
```

I'm trying to deprecate hardcoded references to the above variables.

```suggestion
        AZURE_CLIENT_ID: $(MAPS_CLIENT_ID)
        AZURE_CLIENT_SECRET: $(MAPS_CLIENT_SECRET)
        AZURE_TENANT_ID: $(MAPS_TENANT_ID)
        AZURE_SUBSCRIPTION_ID: $(MAPS_SUBSCRIPTION_ID)
```

---

@benbp Thanks for the reviewing. I am still new to this, I'm not sure what's the difference between the two, and how can I expose the `MAPS_*` variable so the test.yml is able to consume it.

---

@andykao1213 this is all you need to do. Our deployment script will set the `MAPS_*` variables automatically and this allows the default credential builder to work without any modifications.

For various reasons, we don't automatically set the `AZURE_*` variables in the deployment script, because it would break other local testing scenarios when multiple SDKs/tenants are involved.

---

@benbp got it. Thanks for the explanation. Let me change this. Thank you 😃

---

Fixed in 1fd445983f566fc0fe9b553b6141fa559d5d39a7